docs/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+[//]: # (These are comments and are used for reference and will not show up in the PR)
+
+User Requirement:
+
+Description:
+
+Story Points:
+
+Related Issues:
+
+Task List
+
+- [ ] Java Files to  be Added/Modified (Architecture):
+- [ ] UI Tasks (XML):
+- [ ] Database:
+- [ ] Testing (Unit and/or Intent):
+- [ ] Documentation (UML, Storyboard, Mockups, Wiki)
+
```

Comment:
Should have additional information section, like if the issue is a defect could have `Steps to Reproduce` section here.